File: examples/go/crypto/history/getPreviousClose.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/crypto"
)

// Returns the history for the previous close
func GetPreviousClose() {
	cryptoService := crypto.NewCryptoService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := cryptoService.History(&crypto.HistoryRequest{
		Symbol: "BTCUSD",
	})
	fmt.Println(rsp, err)
}

File: examples/js/crypto/history/getPreviousClose.js
const { CryptoService } = require("m3o/crypto");

// Returns the history for the previous close
async function getPreviousClose() {
  let cryptoService = new CryptoService(process.env.MICRO_API_TOKEN);
  let rsp = await cryptoService.history({
    symbol: "BTCUSD",
  });
  console.log(rsp);
}

getPreviousClose();
